OpenSim/Simulation/Model/Geometry.h

```diff
@@ -132,9 +118,12 @@ class OSIMSIMULATION_API Geometry : public Component {
         SimTK::DecorativeGeometry& decoration) const {
```

Comment:
Who is supposed to call `setDecorativeGeometryTransform` and `setDecorativeGeometryAppearance`?
Are they intended to be part of the public interface? I also do not understand what is their intended purpose.

In `setDecorativeGeometryTransform` where does the Array of decorations come from? Is this supposed to be a helper to help add to the decorations Array that generateDecorations is supposed to fill in?

The comment for `setDecorativeGeometryAppearance` is confusing. Manage A by applying A?
Shouldn't the push down to the decorative geometry be completely hidden from the user API? It is not clear that the passed in decoration is being updated according to the Appearance of this Geometry.

Could the underlying SimTK::DecoratievGeometry array representing this OpenSim::Geometry be updated automatically without exposing these or similar methods?